po/fr.po

```diff
@@ -1,53 +1,55 @@
 # French translations for com.github.bytepixie.snippetpixie package.
-# Copyright (C) 2018 THE com.github.bytepixie.snippetpixie'S COPYRIGHT HOLDER
+# Copyright (C) 2019 THE com.github.bytepixie.snippetpixie'S COPYRIGHT HOLDER
 # This file is distributed under the same license as the com.github.bytepixie.snippetpixie package.
-# Automatically generated, 2018.
+# NathanBnm, 2019.
 #
 msgid ""
 msgstr ""
 "Project-Id-Version: com.github.bytepixie.snippetpixie\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2018-04-24 00:44+0100\n"
-"PO-Revision-Date: 2018-04-24 00:44+0100\n"
-"Last-Translator: Automatically generated\n"
-"Language-Team: none\n"
+"PO-Revision-Date: 2019-01-17 20:49+0100\n"
+"Last-Translator: NathanBnm\n"
+"Language-Team: Français\n"
 "Language: fr\n"
 "MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=ASCII\n"
+"Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n > 1);\n"
 
 #: data/com.github.bytepixie.snippetpixie.appdata.xml.in:7
 #: data/com.github.bytepixie.snippetpixie.desktop.in:3 src/MainWindow.vala:29
 msgid "Snippet Pixie"
-msgstr ""
+msgstr "Snippet Pixie"
 
 #: data/com.github.bytepixie.snippetpixie.appdata.xml.in:8
 msgid "Your little snippet helper."
-msgstr ""
+msgstr "Votre petit assistant de raccourcis."
 
 #: data/com.github.bytepixie.snippetpixie.appdata.xml.in:10
 msgid ""
 "Save your often used snippets and then expand them whenever you type their "
 "abbreviation."
 msgstr ""
+"Sauvegardez vos raccourcis souvent utilisés et développez-les à chaque fois que vous tapez leur "
+"abréviation."
 
 #: data/com.github.bytepixie.snippetpixie.appdata.xml.in:11
 msgid "For example:- \"spr`\" expands to \"Snippet Pixie rules!\""
-msgstr ""
+msgstr "Par exemple : - « spr` » s'étendra en « Snippet Pixie rules ! »"
 
 #: data/com.github.bytepixie.snippetpixie.appdata.xml.in:57
 msgid "Byte Pixie"
-msgstr ""
+msgstr "Octet Pixie"
```

Comment:
This is a company name, so should remain as "Byte Pixie".

Reply:
@ianmjones I thought the translation didn't make sense. I made the changes.